engine/src/main/java/org/terasology/config/flexible/FlexibleConfigManager.java

```diff
@@ -18,11 +18,11 @@
 import org.terasology.engine.SimpleUri;
 
 public interface FlexibleConfigManager {
-    void addFlexibleConfig(SimpleUri flexibleConfigUri, FlexibleConfig flexibleConfig);
+    void addConfig(SimpleUri configUri, FlexibleConfig config) throws RuntimeException;
```

Comment:
Sorry if I said otherwise, but this configURIs are really configIDs, so I'd rename all instances of configUri to configId, to stick to what it is rather than its type.